crates/sui-core/src/authority_aggregator.rs

```diff
@@ -1538,12 +1703,12 @@ where
                 tx_digest = ?tx_digest,
                 "Found an effect with good stake over threshold"
             );
-            return cert.verify(&self.committee);
+            return Ok(cert);
         }
 
         // If none has, fail.
-        Err(SuiError::QuorumFailedToExecuteCertificate {
-            errors: state.errors,
+        Err(QuorumExecuteCertificateError {
+            errors: aggregate_errors(&state.errors),
```

Comment:
Can we include the total stake here, so we can figure out whether we have a quorum for a particular error?

Reply:
the return type of `aggregate_errors` is `BTreeMap<String, (Vec<AuthorityName>, StakeUnit)>` where the `StakeUnit` is the sum of stakes who has that error

Reply:
Maybe I'm missing a detail here -- I was thinking that in order to figure out that the total stake in errors is strictly greater than `F`, we would need to know what the total was.  Is there another way to figure that out?

Reply:
I see what you are saying, it's a good point. We could get total stake from quorum driver which has `Arc<AuthorityAggregator>` so a total_stake is accessible, but it's possible that the quorum driver would be reconfigured and the committee/stake distribution changed. I will update the code to attach total stake as well